Modules/xASL_module_Population.m

```diff
@@ -247,7 +244,17 @@
         % Iterate over atlases
         x.dir.dirAtlas = fullfile(x.opts.MyPath, 'external', 'Atlases');
 
-        for iAtlas=1:length(x.S.Atlases)        
+        for iAtlas=1:length(x.S.Atlases)
+            % Note that the number of ROI atlases here should be the same the number of tissue masking chosen
+            % If needed, an atlas or tissue type can be provided multiple times in different combinations
+
+            % We use the specified tissue type
+            % 'GM' = gray matter
+            % 'WM' = white matter
+            % 'WB' = whole brain (= GM+WM)
+            x.S.bTissueMasking = x.S.TissueMasking(iAtlas);

```

Comment:
Now I get it. bTissueMasking is the single instance. Then we should still change the name,

Reply:
Fixed in a7370150eb9b7101ef9e47766dabb1ef1b82e7ad by changing:
xS.bTissueMasking->xS.TissueMaskingLocal